### DISCUSSION TITLE
Groups to Tabs in Reorder

### DISCUSSION THREAD

**Discussion by craigs100** [umbraco/Umbraco-CMS#12499] [Features and ideas]
To ease the transition from a Group-based docType to a Tab-based docType could it be possible, when in "Reorder" mode, to drag the Group title onto the Top bar (where "Generic" lives) to just create a tab with it's contents? It would be much easier than the current situation where you have to create a new tab and then drag each and every group property into the new tab. Half the time it take a dozen goes and so is very time-consuming.

**Comment by skttl** [comment]
I'm pretty sure you can do that already.

**Reply by craigs100** [reply]
WOW! I just opened up a project to check my facts before proving you wrong and discovered the "Convert to Tab" button that appears!  MIND BLOWN! I'd never seen that before and I must be on my 5 or 6 V9 site by now, lol

My days of fiddling about trying to get the right new Tab to highlight so I can drop a property in are now over :)

**Comment by TwoMoreThings** [comment]
I couldn't move an individual property to the Generic tab, nor move the property out of the group once the group was moved. Also, once the property failed to move, the group couldn't be moved unless I stopped reordering, then started again. This was a fresh install of 9.5.0, but it's also the same in 9.3 and 10.0-RC1. Seb was able to see the same result in a conversation we had on Discor.  It doesn't work for me with a property on it's own, also, once I move a group to the generic tab you don't have the drop zone where you can drop the property outside of a group that you have within other tabs

https://user-images.githubusercontent.com/13945979/170862375-84d8dae5-7adf-4425-bd17-0f94a3ba111b.mp4

.

**Reply by craigs100** [reply]
Maybe a similar button to "Convert to Tab" could be provided to "Convert to Group". Might that help?  (never thought I'd see myself typing that!) lol
